src/main/java/subway/path/application/PathService.java

```diff
@@ -15,33 +15,28 @@
 
 @Service
 @RequiredArgsConstructor
-public class PathService {
+public class PathService {  // TODO
```

Comment:
TODO 주석은 제거해 주시면 좋을 것 같아요 😃 
```suggestion
public class PathService {
```